azure-pipelines.yml

```diff
@@ -0,0 +1,18 @@
+trigger:
+  branches:
+    include: ["master", "rel/*"]
+  paths:
+    exclude: ["*.md"]
+
+variables:
+  # TreatWarningsAsErrors: true
```

Comment:
>#  [](start = 2, length = 2)

Nit: is this commented out? If so, do we need this line at all?

Reply:
I aspire to turn it back on someday, but I can remove it for now, since it seems like MAT warnings are impossible to suppress. :)

Reply:
No worries, as long as you meant to leave it in, it's all good.

---
In reply to: [399538253](https://github.com/aspnet/LibraryManager/pull/557#discussion_r399538253) [](ancestors = 399538253)